gulpfile.js/tasks/copy.js

```diff
@@ -16,3 +16,9 @@ gulp.task('copy:design-system', () => {
     .src([`${config.snapshotDir.v4}/**/*`])
     .pipe(gulp.dest(path.join(config.designSystem.dest, 'public')))
 })
+
+gulp.task('copy:repository-yaml', () => {
+  return gulp
+    .src('repository.yaml')
```

Comment:
Can you add this file path to https://github.com/hmrc/assets-frontend/blob/master/gulpfile.js/config.js  with a descriptive key please?

Reply:
Done